libs/ledger/include/ledger/dag/dag.hpp

```diff
@@ -162,21 +161,21 @@ class DAG : public DAGInterface
   std::unordered_map<NodeHash, std::vector<DAGNodePtr>> loose_nodes_lookup_;  // nodes that are missing one or more references (waiting on NodeHash)
   // clang-format on
 
-  // std::unordered_map<NodeHash, uint64_t>                loose_nodes_ttl_;   // TODO(HUT): loose
-  // nodes management scheme
+  // TODO(HUT): loose nodes management scheme
```

Comment:
Can you raise a tracking github issue for this

Reply:
done